app/src/org/commcare/google/services/ads/AdMobManager.java

```diff
@@ -0,0 +1,98 @@
+package org.commcare.google.services.ads;
+
+import android.content.Context;
+import android.view.View;
+import android.widget.FrameLayout;
+
+import com.google.android.gms.ads.AdRequest;
+import com.google.android.gms.ads.AdSize;
+import com.google.android.gms.ads.AdView;
+import com.google.android.gms.ads.MobileAds;
+
+import org.commcare.CommCareApplication;
+import org.commcare.dalvik.BuildConfig;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Controls all actions related to showing ads in consumer apps via AdMob
+ *
+ * @author Aliza Stone
+ */
+public class AdMobManager {
+
+    private static final Map<String, String> adMobIdsMap = initAdMobIdsMap();
+
+    public static final String CLINICAL_SCALES_PACKAGE_ID =
```

Comment:
@Aliza - Confirming that we can't include these through some external configuration file? 

Gayatri and I are looking to run tests with ~4 consumer apps in the next week or two, do we need to try to get all of those ids over to you before this gets pulled?